app/views/checkout/thankyou.scala.html

```diff
@@ -23,14 +23,15 @@
 <script type="text/javascript">
     guardian.pageInfo.slug="GuardianDigiPack:Order Complete";
     guardian.pageInfo.productData = {
-        // Omniture - TODO - fix the type field
+        numberOfMonths: @{plan.billingPeriod.numberOfMonths},
+        // Omniture
         source: 'Subscriptions and Membership',
         type: 'GUARDIAN_DIGIPACK',
```

Comment:
I won't fix the type field as I have been given no advice on what string to use for the other products.